simple/php/extract-variable_before2.php

```diff
@@ -0,0 +1,10 @@
+void renderBanner() {
+  // When all the parameters are set, counted 
+  // the boundaries of the frame, we print the 
+  // result on the screen.
+  if ((($frame->isChanged() or $target->isChanged()) or 
+        $experiment->isRunning()) and
+        $frame->getSize()->equal($screen->getSize())) {
```

Comment:
This is more natural comparison in PHP world:
$frame->getSize() == $screen->getSize()